xAPI.md

```diff
@@ -414,7 +415,7 @@ examples of activities include a book, an e-learning course, a hike or a meeting
 __Activity Provider (AP)__: The software object that is communicating with 
 the LRS to record information about a learning experience. May be similar to a SCORM 
 package in that it is possible to bundle learning assets with the software object that performs this 
-communication, but an Activity Provider could also be separate from the experience it is reporting about.
+communication, but an Activity Provider could also be separate from the experience it is reporting about. This term is deprecated and will be replaced by ["Statement Provider"](#def_statement_provider) in future editions of the xAPI.
```

Comment:
I agree that changing names after a lot of people are already using existing names can lead to problems.  However, I believe that this needs to be balanced with the need to provide terminology which accurately reflects the terms being discussed.  The essential function of activity providers is not to provide activities but that to provide statements of learning experience to the learning record stores.  

I think that the xAPI is relatively early in it's roll out and I don't think it would confuse those who currently use the term 'activity provider' any more than replacing the term 'Indian' with  'Native American' has confused who used the replaced term about who is being referred to.